packages/core/src/common/dom/set-unique-element-id.ts

```diff
@@ -13,7 +13,10 @@ export default function setUniqueElementId(selector: string, idPrefix: string):
 
     return Array.prototype.slice.call(containers).map((container: HTMLElement) => {
         if (!container.id) {
-            container.id = uniqueId(idPrefix);
+            const getUniqId = (id: string): string =>
+                document.getElementById(id) ? getUniqId(uniqueId(idPrefix)) : id;
+
+            container.id = getUniqId(uniqueId(idPrefix));
```

Comment:
Can we put `uniqueId(idPrefix)` inside the `getUniqId` function so that `uniqueId(idPrefix)` only appears once in the code?